api/lib/infrastructure/repositories/user-repository.js

```diff
@@ -432,6 +432,21 @@ module.exports = {
         }
         throw err;
       });
-  }
+  },
+
+  async updateUserAttributes(id, userAttributes) {
```

Comment:
Question : Faut-il inclure dans cette méthode un _omit_ d'un éventuel `userAttributes.id` ?

Reply:
Vu ensemble, c'est un nice to have, ca marche pour les deux cas.